pkg/convert/convert.go

```diff
@@ -213,14 +213,15 @@ func GetRelationsRequestToV3(in *dsr2.GetRelationsRequest) *dsr3.GetRelationsReq
 	)
 
 	return &dsr3.GetRelationsRequest{
-		ObjectType:      objType,
-		ObjectId:        in.GetParam().GetObject().GetKey(),
-		Relation:        in.GetParam().GetRelation().GetName(),
-		SubjectType:     in.GetParam().GetSubject().GetType(),
-		SubjectId:       in.GetParam().GetSubject().GetKey(),
-		SubjectRelation: "",
-		WithObjects:     false,
-		Page:            PaginationRequestToV3(in.Page),
+		ObjectType:               objType,
+		ObjectId:                 in.GetParam().GetObject().GetKey(),
+		Relation:                 in.GetParam().GetRelation().GetName(),
+		SubjectType:              in.GetParam().GetSubject().GetType(),
+		SubjectId:                in.GetParam().GetSubject().GetKey(),
+		SubjectRelation:          "",
+		WithEmptySubjectRelation: true,
```

Comment:
This ensures that calls to the v2 GetRelations only return results that do not have subject relations.